yarn.lock

```diff
@@ -2115,11 +2115,6 @@ camelcase@^2.0.0:
   resolved "https://registry.yarnpkg.com/camelcase/-/camelcase-2.1.1.tgz#7c1d16d679a1bbe59ca02cacecfb011e201f5a1f"
   integrity sha1-fB0W1nmhu+WcoCys7PsBHiAfWh8=
 
-camelcase@^3.0.0:
```

Comment:
No changes to the `package.json` but still changes in here...? 🤔

Reply:
Strange...